Homepage-options/Landing - redesign-Rohit/style.css

```diff
@@ -0,0 +1,502 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+/* The header part */
+.header {
+    height: 95vh;
+    background-image: linear-gradient(
+        to right bottom,
+        rgba(51, 51, 51, 0.438),
+        rgba(34, 34, 34, 0.411)
+      ),
+      url(../img/hero.jpg);
+    background-size: cover;
+    background-position: top;
+    position: relative;
+    clip-path: polygon(0% 0%, 100% 0%, 100% 75vh, 0% 100%);
+  }
+
+.header_logo_box {
+    position: absolute;
+    top: 4rem;
+    left: 4rem;
+    mix-blend-mode: color-burn;
+    
+}
+
+.header_logo {
+    width: 4rem;
+
+}
+
+
+.header__text-box {
+    position: absolute;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+    width: 100%;
+    
+  }
+
+
+.heading-primary {
+    color: #fff;
+    text-transform: uppercase;
```

Comment:
The content "uppercase/lowercase' should be from the content and not from the CSS.